### PR TITLE
Updating Chat and Chat members docs to support App context

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

File: api-reference/beta/api/chat-get-members.md [hunk 1]
```diff
@@ -23,7 +28,7 @@ One of the following permissions is required to call this API. To learn more, in
 |---------|-------------|
 |Delegated (work or school account)| ChatMember.Read, ChatMember.ReadWrite, Chat.ReadBasic, Chat.Read, Chat.ReadWrite |
 |Delegated (personal Microsoft account)|Not supported.|
-|Application| Not supported. |
+|Application| ChatMember.Read.All, ChatMember.ReadWrite.All, Chat.ReadBasic.All, Chat.Read.All, Chat.ReadWrite.All. |
```
Comment (Lauragra) [inline api-reference/beta/api/chat-get-members.md:31]: Is support for application permissions a change that you want to call out in the changelog?

File: api-reference/beta/api/chat-get.md [hunk 2]
```diff
@@ -53,7 +53,43 @@ Do not supply a request body for this method.
 
 If successful, this method returns a `200 OK` response code and a collection of [chat](../resources/chat.md) objects in the response body.
 
-## Example
+## Examples
+
+### Example 1 : Get a group chat
+##### Request
```
Comment (Lauragra) [inline api-reference/beta/api/chat-get.md:59]: The heading levels should be sequential so this should be an H4 ####. Also, please remove the space before the colon.

File: api-reference/beta/api/chat-get.md [hunk 3]
```diff
@@ -111,6 +145,103 @@ Content-length: 201
 }
 ```
 
+### Example 3 : Get a chat and all it's members
```
Comment (Lauragra) [inline api-reference/beta/api/chat-get.md:148]: it's -> its

**Review by bhartono — APPROVED** [review]

**Comment by bhartono** [comment]
dont forget to add changelog.json

**Review by Lauragra — APPROVED** [review]

**Pull request merged**